### PR TITLE
docs: fix shadow variable in tutorial example

### PR DESCRIPTION
**Pull request by Hearen**
Avoid local variable shadowed by the parameter hero

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by trotyl** [comment]
The scope should be `docs`, code examples are not part of the framework, but of the documentation.

File: aio/content/examples/toh-pt6/src/app/hero.service.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ export class HeroService {
   /** POST: add a new hero to the server */
   addHero (hero: Hero): Observable<Hero> {
     return this.http.post<Hero>(this.heroesUrl, hero, httpOptions).pipe(
-      tap((hero: Hero) => this.log(`added hero w/ id=${hero.id}`)),
+      tap((newHero: Hero) => this.log(`added hero w/ id=${newHero.id}`)),
```
Comment (brandonroberts) [inline aio/content/examples/toh-pt6/src/app/hero.service.ts]: To keep consistent with the `updateHero` method.

```
tap(_ => this.log(`added hero w/ id=${hero.id}`)),
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Update git commit message to `docs: fix shadow variable in tutorial example`

**Comment by brandonroberts** [comment]
Closed for https://github.com/angular/angular/pull/28499

**Pull request closed without merging**